backend/tracim_backend/models/context_models.py

```diff
@@ -205,14 +205,27 @@ def __init__(self, acp: str):
         self.acp = acp
 
 
+class FileQuery(object):
```

Comment:
What is the point of creating `FileQuery` class?

Reply:
Allow turning marshmallow schema into object, Like most object in  backend/tracim_backend/views/core_api/schemas.py